ljps-frontend/pages/learning-journey/view-skills/index.js

```diff
@@ -13,9 +13,10 @@ export default function ViewSkills() {
     roleDepartment: "",
     roleDescription: "",
   });
+  const [checked, setChecked] = useState(false);
```

Comment:
Remove unused state

Reply:
Roger! My apologies for missing that out